### ISSUE TITLE
Finetune on custom dataset?

### ISSUE THREAD

**Issue by DesaleF** [KaihuaTang/Scene-Graph-Benchmark.pytorch#174]
Hi, Dear Author!
First thank you for sharing your work. I explore the code and also tested the model on random images. I found your model very useful for my project. However, I have object labels and relationship(predicates) which doesn't exist in visual-gnome dataset. So I decided to finetune your model using my custom dataset. Can you help me with the steps I should follow to use your code for my custom dataset?

Thank you!

**Comment by bibekyess** [comment]
Hi @DesaleF! Did you figure out how to do it? I can retrain the custom dataset from scratch but did you figure out how to fine tune it on the top of pretrained model (on VG dataset)?

**Comment by bibekyess** [comment]
Maybe this helps: #180

**Comment by majianbo3** [comment]
> Maybe this helps: #180

Hello, I have been learning about scene maps recently, but I am not sure how to create my own training set. If you know how to create it and let me know, I would be very grateful

**Comment by zzzcccxx** [comment]
> > Maybe this helps: #180
> 
> Hello, I have been learning about scene maps recently, but I am not sure how to create my own training set. If you know how to create it and let me know, I would be very grateful

Hello, do you now know how to create  dataset?
